### ISSUE TITLE
Adding a team section to the README

### ISSUE THREAD

**Issue by apmoore1** [UCREL/pymusas#11]
I was wondering if we should add a Team section to the README this is based on reading the README from the AllenNLP GitHub repository whereby they have a [team section](https://github.com/allenai/allennlp/blob/main/README.md#team).

The team section could be something small like:

# Team

PyMUSAS is an open-source project that has been created and funded by the [University Centre for Computer Corpus Research on Language](https://ucrel.lancs.ac.uk/) (UCREL) at [Lancaster University](https://www.lancaster.ac.uk/). For more information on who has contributed to this code base see the [contributions page.](https://github.com/UCREL/pymusas/graphs/contributors) 

Instead of calling it `Team` may be call it `Team and Funding`?

**Comment by perayson** [comment]
yes, definitely a good idea, and you should be mentioned there. If you add that then I can have a look and tweak next week.
